console/module/authorization/src/main/resources/org/eclipse/kapua/app/console/module/authorization/client/messages/ConsolePermissionMessages.properties

```diff
@@ -152,3 +152,21 @@ dialogDeleteRoleInfo=Delete the association between the user and the role?
 dialogDeleteRoleConfirmation=Role association deleted successfully.
 dialogDeleteRoleError=User association delete failed: {0}
 
+filterFieldRoleNameLabel=Name
```

Comment:
I'm not sure of this. 
Is that right that some of these `*Messages.properties` entries have the same value of the  `ConsoleRoleMessages.properties`??